common/src/main/java/com/alibaba/nacos/common/remote/client/RpcConstants.java

```diff
@@ -33,64 +33,121 @@
  */
 public class RpcConstants {
     
-    public static final String NACOS_CLIENT_RPC = "nacos.remote.client.rpc";
+    public static final String NACOS_SDK_CLIENT_RPC = "nacos.remote.sdk.client.rpc";
```

Comment:
I just feel that this might be better for differentiation. If we consider compatibility, the default would be the SDK, and the one with the cluster would be specifically for the cluster. I'll make some adjustments later.

Reply:
done.